Makefile

```diff
@@ -303,6 +306,15 @@ golangci-lint: $(GOLANGCI_LINT)
 .PHONY: lint-go
 lint-go: golangci-lint ## Run all code checks for backend. You can use GO_LINT_FILES to specify exact files to check
 
+.PHONY: lint-go-diff
+lint-go-diff: $(GOLANGCI_LINT)
+	git diff --name-only remotes/origin/main | \
```

Comment:
nit: this won't work when targeting other branches: eg for backports:
it would be nice to set it during runtime

Reply:
That's a very good point :thinking:
Do you think a `BASE` variable defaulting to `remotes/origin/main` could work? Or would there be a better way of detecting the base (looked up but couldn't find)?

Reply:
Merging, but I'm interested in following up, let's catch up async